### ISSUE TITLE
Possible bug in train validation split

### ISSUE THREAD

**Issue by infoshoc** [anuragranj/coma#5]
https://github.com/anuragranj/coma/blob/643b23474659743b13839a05818873256464df2a/facemesh.py#L32-L34

Is it possible there is some bug: like you wanted to take first n - nVal meshes as train and last nVal as validation and actually throwed away last nVal and taking first n - nVal as train and n-2nVal to nVal as validation?

I believe the fix is:
```
		vertices_train = np.load(self.train_file)
		self.vertices_train = vertices_train[:-self.nVal]
		self.vertices_val = vertices_train[-self.nVal:]
```

Prove me, if I am wrong.

**Comment by anuragranj** [comment]
I believe this is what I wanted.
```python
elements = [1,2,3,4,5,6,7,8,9,10]
nVal= 2
train = elements[:-nVal]
val = elements[-nVal:]
train
Out: [1, 2, 3, 4, 5, 6, 7, 8]
val
Out: [9, 10]
```

**Comment by infoshoc** [comment]
But what you are actually doing in code, as far as I understand is 
```

train = [1,2,3,4,5,6,7,8,9,10]
nVal= 2
train = train[:-nVal]
val = train[-nVal:]
train
Out: [1, 2, 3, 4, 5, 6, 7, 8]
val
Out: [7, 8]


```

not very critical

**Comment by anuragranj** [comment]
Oh! I see your point. Fixed in cf0509b8ff6dee50431bc102f969b6f2ac2bb772.
Thanks.
